### PR TITLE
Switch from deprecated EnableGlobalMethodSecurity to EnableMethodSecu…

### PR DESCRIPTION
**Pull request by avgustinmm**
…rity

### CODE REVIEW

File: hawkbit-rest-core/src/main/java/org/eclipse/hawkbit/rest/SecurityManagedConfiguration.java [hunk 1]
```diff
@@ -39,7 +37,6 @@
 @Slf4j
 @Configuration
 @EnableWebSecurity
-@EnableGlobalMethodSecurity(prePostEnabled = true, mode = AdviceMode.ASPECTJ, proxyTargetClass = true, securedEnabled = true)
```
Comment (vasilchev) [inline hawkbit-rest-core/src/main/java/org/eclipse/hawkbit/rest/SecurityManagedConfiguration.java:42]: isnt here a change in behavior ? mode = AdviceMode.ASPECTJ wheres by default mode is PROXY ?

Comment (avgustinmm) [reply hawkbit-rest-core/src/main/java/org/eclipse/hawkbit/rest/SecurityManagedConfiguration.java:42]: @PreAuthenticate annotations are used only in repository and management API
that's why the EnableMethod security is moved only in repository and management API configurations

Comment (avgustinmm) [reply hawkbit-rest-core/src/main/java/org/eclipse/hawkbit/rest/SecurityManagedConfiguration.java:42]: and regarding removal of ASPECTJ - it seems this mode doesn't work - and it was the server app annotations with  PROXY mode that really makes things to work.

**Review by vasilchev — APPROVED** [review]
lgtm

**Pull request merged**